nwamugo/politico-react
# ft-user-can-sign-up #166333191

What does this PR do?
It implements user can sign up

Description of Task to be completed?
User can sign in to the app Politico

How should this be manually tested?
Visit https://nwamugo-politico.netlify.com/
And click sign up

What are the relevant pivotal tracker stories?(if applicable)
#166333191 Users should be able Signup to Politico
